src/app/onion/dashboard/dashboard.component.ts

```diff
@@ -2,28 +2,52 @@ import { Component, OnInit } from '@angular/core';
 import { HistoryService } from 'app/core/history.service';
 import { NavigationEnd, Router } from '@angular/router';
 import { NavbarService } from 'app/core/navbar.service';
+import { LearningObject } from '@entity';
+import { LearningObjectService } from 'app/onion/core/learning-object.service';
+import { AuthService } from 'app/core/auth.service';
 import { Subject } from 'rxjs';
 
 @Component({
   selector: 'clark-dashboard',
   templateUrl: './dashboard.component.html',
-  styleUrls: ['./dashboard.component.scss'],
+  styleUrls: ['./dashboard.component.scss']
 })
-export class DashboardComponent {
+export class DashboardComponent implements OnInit {
   lastLocation: NavigationEnd;
   activeIndex = 0;
+  loading: boolean;
+  releasedLearningObjects: LearningObject[];
+  workingLearningObjects: LearningObject[];
 
   action$: Subject<number> = new Subject();
 
+  filters: any;
+
   constructor(
     private history: HistoryService,
     private router: Router,
-    private navbar: NavbarService
+    private navbar: NavbarService,
+    private learningObjectService: LearningObjectService,
+    public auth: AuthService
   ) {
     this.navbar.hide();
     this.lastLocation = this.history.lastRoute;
   }
 
+  async ngOnInit() {
+    this.loading = true;
+    // retrieve draft status learning objects
+    setTimeout(async() => {
+      this.workingLearningObjects = await this.getDraftLearningObjects(
+        {status: ['unreleased', 'proofing', 'review', 'rejected', 'waiting']}
```

Comment:
What was the reasoning behind filtering draft Learning Objects by all statuses instead of providing no filters if none are specified by the user?

By default, when the `draftsOnly` filter is specified, that included Learning Objects in all of the statuses listed.